conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureUndertowRuntime.java

```diff
@@ -132,8 +141,51 @@ public Builder requestArgHandler(RequestArgHandler value) {
             return this;
         }
 
+        @CanIgnoreReturnValue
+        public Builder onSetRequestToken(BiConsumer<HttpServerExchange, Optional<UnverifiedJsonWebToken>> callback) {
+            onSetRequestToken = callback;
+            return this;
+        }
+
         public ConjureUndertowRuntime build() {
             return new ConjureUndertowRuntime(this);
         }
     }
+
+    private static final class OnSetRequestTokenAuthExtractor implements AuthorizationExtractor {
```

Comment:
We can pass our handler into the ConjureAuthorizationExtractor, no need to reimplement AuthorizationExtractor here, this way we can avoid dealing with `Attachments.UNVERIFIED_JWT`

Reply:
fixed